### PR TITLE
C++ resource for competitive programming

### PR DESCRIPTION
**Pull request by kshitijmishra27**
Added a C++ Standard Template Library youtube video specially for competitive programming .
CP(competitive programming) is a trending programming sport in India where people mainly use C++ for CP and there is a lot of usage of C++ STL for competitive programming.
 So I added a one shot video by Raj vikram aditya ( a Google SDE and a famous competitive programmer of india) who taught about STL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [C++ Tutorial](https://www.youtube.com/watch?v=DamuE8TM3xo&list=PLGLfVvz_LVvQ9S8YSV0iDsuEU8v11yP9M)
 - :file_folder: [C++ API Reference](https://www.cplusplus.com/reference/)
 - :video_camera: [C++ Standard Template Library in Practice](https://www.udemy.com/course/c-standard-template-library-in-practice/learn/)
-
+- :video_camera: [C++ Standard Template Library for Competative programming](https://www.youtube.com/watch?v=RRVYpIET_RU)
```
Comment (shahednasser) [inline README.md:196]: Can you add a new empty line after this?

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kshitijmishra27) [reply README.md:196]: shure

Comment (shahednasser) [reply README.md:197]: After not before

**Comment by shahednasser** [comment]
Thank you!

**Pull request merged**